packages/desktop-client/src/components/util/GenericInput.js

```diff
@@ -98,6 +102,28 @@ export default function GenericInput({
       }
       break;
 
+    case 'id2':
```

Comment:
Why couldn’t `id` work here?

Reply:
The id label gives `is/contains/one of` options which doesn't work for the saved condition. Might be a good idea to change it to a better more intuitive name?

Reply:
Updated the naming convention to better reflect the new type.